### DISCUSSION TITLE
the project has a virus

### DISCUSSION THREAD

**Discussion by 0vertrace** [community/community#124568] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

https://github.com/Chrisaacosta/Solara-Executor there is a virus

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by davevad93** [comment]
Hi @pav-coder ,

If you have evidence that a repo contains malware you can report it, to do it look to the right side where the `About` section is displayed, click on `Report repository` and then select the `Active Malware or Exploits` category.

![0](https://github.com/community/community/assets/94308234/9276421c-ab2a-496e-9cb8-76bc4409389b)

**Comment by 0vertrace** [comment]
Ok sorry

сб, 18 трав. 2024 р., 23:57 користувач Davide Presti <
***@***.***> пише:

**Reply by davevad93** [reply]
Why are you sorry? You haven't done anything wrong. :smiley:

**Comment by 0vertrace** [comment]
I mean, I made a mistake.🙃

вс, 19 мая 2024 г. в 00:38, Davide Presti ***@***.***>:

**Comment by Mrgrizz3** [comment]
Reported the project and got it deleted now
